shared-version.properties

```diff
@@ -1,4 +1,4 @@
 // Updating blackduck-common version here might necessitate an update to integration-common version in src/main/resources/create-gradle-airgap-script.ftl \
 // to ensure the integration-common versions in both airgap and non-airgap are the same. 
-gradle.ext.blackDuckCommonVersion='66.2.23'
+gradle.ext.blackDuckCommonVersion='replace-me-with-appropriate-blackduck-common-version'
```

Comment:
Can you confirm if my understanding is correct: the main change here is going to be bumping up the BD Common version and the rest is simply a method rename?

Reply:
https://github.com/blackducksoftware/synopsys-detect/pull/1187#discussion_r1693227482